signalfx-org-metrics/README.md.jinja

```diff
@@ -67,21 +67,21 @@ These metrics track limits that Infrastructure Monitoring enforces for your orga
 * `sf.org.limit.hosts` (gauge): Maximum number of hosts that can send data to your organization.
    The limit is higher than your contractual limit to allow for burst and overage usage. If you
```

Comment:
Question: so- to make sure I understand this correctly- let's say the user has limit x. If the user hits a maximum of x + 20 the data still flows in, but  but no data points are added to new hosts? 

Question: Does the user receive a notification when they exceed their limit and drop data points from new hosts? If so- I think it would be helpful to mention that here so that the user understands that as long as they don't receive a notification everything is fine. On the contrary- if there is no notification- there should be! and a good opportunity to bring it up with PM :) 

 Question: As a user, my next question is, so if I go over, what happens? Do I contact sales to increase my limits ? Is there an obvious affordance and link in the UI for this, or something we can mention in the docs on how to take the next step?

Reply:
the above comments/questions apply to all references of the "exceed this limit ...occasional overage..." in the doc 👀

Reply:
Good questions! Data points are accepted for the first x hosts, plus a bit of a fudge as noted. No new hosts get MTS, which means that the data points they sent are dropped.
This documentation describes the _metric_ that provides real-time reports of the limit. Customers don't automatically receive a notification. They have two options:

1. They can set up a custom detector for the metric, and get notified when it exceeds a value.
2. They can set up a quota and threshold on the org tokens for the clients that send data points to IMM. IMM then sends out a notification when a client exceeds the threshold (if set) and when it exceeds the limit.

This description is just a summary. Host limits are only enforced if the customer has host-based pricing. I can explain more over Zoom.